src/metadata/models/metadata.ts

```diff
@@ -0,0 +1,253 @@
+import { Column, Entity, Index, PrimaryColumn } from 'typeorm';
+
+export interface ILink {
+  name?: string;
+  description?: string;
+  protocol: string;
+  url: string;
+}
+
+export interface Base {
```

Comment:
Should this class be exported? it seams like the only usage is inside this file

Reply:
Maybe rename the interface `MetadataBase` or `PayloadBase`
`Base` is a bit too generic

Reply:
Done